source/_docs/configuration/customizing-devices.markdown

```diff
@@ -100,6 +100,9 @@ homeassistant:
       icon: mdi:kettle
     switch.rfxtrx_switch:
       assumed_state: false
+    media_player.my_media_player
```

Comment:
This change is actually incorrect and broken as the YAML is now invalid.

Reply:
I haven't included the : after media_player. I've added this as a suggestion below, I hope that's the correct way to resolve this?
```suggestion
    media_player.my_media_player:
```

Reply:
No, it needs a new PR to be able to resolve, as this one is merged.